napari/_qt/layers/qt_image_layer.py

```diff
@@ -46,22 +47,36 @@ def __init__(self, layer):
         sld.setSingleStep(1)
         sld.setValue(self.layer.iso_threshold * 100)
         sld.valueChanged[int].connect(
-            lambda value=sld: self.changeIsoTheshold(value)
+            lambda value=sld: self.changeisoThreshold(value)
         )
-        self.isoThesholdSilder = sld
-        self.isoThesholdLabel = QLabel('iso threshold:')
+        self.isoThresholdSlider = sld
+        self.isoThresholdLabel = QLabel('iso threshold:')
+
+        sld = QSlider(Qt.Horizontal)
+        sld.setFocusPolicy(Qt.NoFocus)
+        sld.setMinimum(0)
+        sld.setMaximum(200)
+        sld.setSingleStep(1)
+        sld.setValue(self.layer.attenuation * 100)
+        sld.valueChanged[int].connect(
+            lambda value=sld: self.changeAttenuation(value)
+        )
```

Comment:
same here:  how about just `sld.valueChanged.connect(self.changeAttenuation)`?

Reply:
as noted below will defer for followup PR